tools/run_tests/run_xds_tests.py

```diff
@@ -2595,12 +2595,7 @@ def __init__(self, compute, alpha_compute, project, project_num):
         client_env['GRPC_XDS_BOOTSTRAP'] = bootstrap_path
         client_env['GRPC_XDS_EXPERIMENTAL_CIRCUIT_BREAKING'] = 'true'
         client_env['GRPC_XDS_EXPERIMENTAL_ENABLE_TIMEOUT'] = 'true'
-        # Temporarily turn off fault injection, because HTTPFault filter isn't
-        # handled correctly yet in CPP and Go. And setting would break all the
-        # other tests. Uncomment the following line when the support is
-        # complete.
-        #
-        # client_env['GRPC_XDS_EXPERIMENTAL_FAULT_INJECTION'] = 'true'
+        client_env['GRPC_XDS_EXPERIMENTAL_FAULT_INJECTION'] = 'true'
```

Comment:
We can't enable this until Go adds fault injection support, otherwise HTTP filter support will make all tests fail.